changelogs/current.yaml

```diff
@@ -26,6 +26,9 @@ minor_behavior_changes:
 - area: cache_filter
   change: |
     add a completion callback to updateHeaders interface. Any external cache implementations will need to update to match this new interface. See changes to simple_http_cache in PR#23666 for example.
+- area: cache_filter
```

Comment:
nit: I suggest moving it up to `behavior changes`, and explicitly say that "api path of work-in-progress extension changed from..."

Reply:
Added clarity, but I think minor behavior changes is the right category: `# *Changes that may cause incompatibilities for some users, but should not for most*`
(Given that it's an extension that's WIP *and* explicitly not production ready, it seems like even "some users" should be an overestimate of the impact.)